lib/spoom/sorbet/lsp.rb

```diff
@@ -11,10 +11,12 @@
 module Spoom
   module LSP
     class Client
-      def initialize(sorbet_cmd, *sorbet_args)
+      def initialize(sorbet_cmd, *sorbet_args, path: ".")
         @id = 0
         Bundler.with_clean_env do
-          @in, @out, @err, @status = Open3.popen3([sorbet_cmd, *sorbet_args].join(" "))
+          opts = {}
+          opts[:chdir] = path
```

Comment:
```suggestion
          opts = { chdir: path }
```

?

Reply:
This one is to please Sorbet which has a hard time figuring how to pass the hash:

https://sorbet.run/#%23%20typed%3A%20true%0A%0AOpen3.capture3(%22ls%22%2C%20%7Bchdir%3A%20%22.%22%7D)%0A%0Aopts%20%3D%20%7Bchidr%3A%20%22.%22%7D%0AOpen3.capture3(%22ls%22%2C%20opts)%0A%0Aopts%20%3D%20%7B%7D%0Aopts%5B%3Achidr%5D%20%3D%20%22.%22%0AOpen3.capture3(%22ls%22%2C%20opts)

 This is a known bug and they recently started working on it, should be resolved once we implement Ruby2.7 kargs changes correctly.